grype/presenter/cyclonedx/document.go

```diff
@@ -69,7 +73,21 @@ func NewVulnerability(m match.Match, p vulnerability.MetadataProvider) (Vulnerab
 	}
 
 	rating.Score = score
-	rating.Severity = metadata.Severity
+
+	r := Ratings{
+		Rating: []Rating{rating},
+	}
+
+	// The schema does not allow "Negligible", only allowing the following:
```

Comment:
@wagoodman there is some disparity here, not sure why `Negligible` is not allowed by the schema but captured in the database

Reply:
The DB build process allows any values from upstream and does not do any validation around this --upstream is the source of truth on "correctness". Upstream "Negligible" is valid, it happens to be in CycloneDX specifically downstream where this is not allowed. It may be tempting to change upstream (in the DB build process) to eliminate this discrepancy, however, this would implicitly couple CycloneDX's opinions with what is "valid" in the DB, which feels wrong --presentation concerns/opinions/restrictions should say with presenters.

Reply:
That's right, this is CycloneDX being opinionated about this being invalid. I checked and other systems like Github's SARIF report allows it, so I vote to just mangle it here for Cyclone

Reply:
Just a note for the sake of clarity: this ended up coming from Anchore itself, where it produces "Negligible". CycloneDX does not allow it because it isn't part of the allowed ratings as per https://nvd.nist.gov/vuln-metrics/cvss